examples/GitHubLovesSparkDotNet/Program.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Spark.Sql;
+using static Microsoft.Spark.Sql.Functions;
+using System.Text.RegularExpressions;
+
+namespace GitHubLovesSparkDotNet
+{
+    // Application presented at Microsoft //Build 2019:
+    // 1. By Scott Hanselman & Scott Hunter in the following session:
+    //    BRK3015 - .NET Platform Overview and Roadmap
+    //    https://mybuild.techcommunity.microsoft.com/sessions/77031?source=sessions
+    // 2. By Rahul Potharaju in the following session:
+    //    BRK3055 - Building data pipelines for Modern Data Warehouse with Spark and .NET in Azure
+    //    https://mybuild.techcommunity.microsoft.com/sessions/76996?source=sessions
+    //
+    // This app is purely for demonstration purposes. And more 
+    // importantly, this was not written by a seasoned Data Scientist 
+    // so please avoid deriving *any* conclusions from the results :) 
+    //
+    // Of course, please feel free to open PRs to improve and make 
+    // this app more interesting.
+    //
+    // Acknowledgements:
+    // - The GHTorrent project
+    //   http://ghtorrent.org
+    // - Azure Databricks 
+    //   https://azure.microsoft.com/en-us/services/databricks/
+    // - Azure Data Lake Storage Gen2 (ABFS) 
+    //   https://azure.microsoft.com/en-us/services/storage/data-lake-storage/
+
+    public class Program
+    {
+        public const string StorageConfigKey = "fs.azure.account.key.<your-storage-account-name>.dfs.core.windows.net";
+        public const string SecureStorageKey = "<your-storage-account-key>";
+        public const string DataStoragePath = "abfss://<filesystem>@<storage-account>.dfs.core.windows.net/<path>/";
+
+        static void Main(string[] args)
+        {
+            SparkSession spark = SparkSession
+                    .Builder()
+                    .Config(StorageConfigKey, SecureStorageKey)
+                    .AppName(@"Github 💖 .NET for Apache Spark")
+                    .GetOrCreate();
+
+            // Initialize all dataframes (which point to CSV files on the storage system)
+            DataFrame commits = 
+               spark.Read()
```

Comment:
I meant the same style throughout this file.

Reply:
Done